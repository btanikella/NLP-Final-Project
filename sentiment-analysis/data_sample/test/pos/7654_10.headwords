tapestry
film
expect
story
is
get
And
