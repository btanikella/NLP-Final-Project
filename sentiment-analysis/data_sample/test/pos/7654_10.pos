DT NN VBZ DT NN  DT NN IN NNS IN NN NNS VBN RB IN NN
PRP VBZ RB RB DT NN POS NN  IN IN DT NN IN VBN NN : CC WP NN EX VBZ  VBZ RB JJ TO DT NN IN DT NN
DT MD VB DT NNP NNP NN IN  NNS  NN  CC NN  CC DT NN VBG DT JJ JJ NNS : VBP RB VB IN DT NN VBZ IN PRP NNS DT NN IN DT NN IN NNS  RB IN VBG DT NN IN NNS
DT NN VBZ DT NN IN NN -LRB- NNP TO NNP -RRB- CC : NN -LRB- PRP MD RB VB -RRB- IN DT NNS
EX VBZ DT NN IN NN
PRP VBP DT NN IN DT JJ NNS  RB VBN IN JJ NNS WRB JJ NN VBZ RB JJ
CC DT JJ NN IN NN IN NNS  IN PRP$ NN IN NN IN NNP PRP VBZ DT NN TO NN NN  RB RB VBN  IN DT JJ NN IN NN NN  CC DT NN IN DT NN WDT VBZ DT NN DT NNS VBP VBN  CC RB NN
