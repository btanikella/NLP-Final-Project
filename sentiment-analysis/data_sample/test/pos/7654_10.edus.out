Sentence #1 (18 tokens):
This film is a tapestry, a series of portraits of Rom communities woven together by music.
[Text=This CharacterOffsetBegin=4 CharacterOffsetEnd=8 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=9 CharacterOffsetEnd=13 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=14 CharacterOffsetEnd=16 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=17 CharacterOffsetEnd=18 PartOfSpeech=DT] [Text=tapestry CharacterOffsetBegin=19 CharacterOffsetEnd=27 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=27 CharacterOffsetEnd=28 PartOfSpeech=,] [Text=a CharacterOffsetBegin=29 CharacterOffsetEnd=30 PartOfSpeech=DT] [Text=series CharacterOffsetBegin=31 CharacterOffsetEnd=37 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=38 CharacterOffsetEnd=40 PartOfSpeech=IN] [Text=portraits CharacterOffsetBegin=41 CharacterOffsetEnd=50 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=51 CharacterOffsetEnd=53 PartOfSpeech=IN] [Text=Rom CharacterOffsetBegin=54 CharacterOffsetEnd=57 PartOfSpeech=NN] [Text=communities CharacterOffsetBegin=58 CharacterOffsetEnd=69 PartOfSpeech=NNS] [Text=woven CharacterOffsetBegin=70 CharacterOffsetEnd=75 PartOfSpeech=VBN] [Text=together CharacterOffsetBegin=76 CharacterOffsetEnd=84 PartOfSpeech=RB] [Text=by CharacterOffsetBegin=85 CharacterOffsetEnd=87 PartOfSpeech=IN] [Text=music CharacterOffsetBegin=88 CharacterOffsetEnd=93 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=93 CharacterOffsetEnd=94 PartOfSpeech=.] 
root(ROOT-0, tapestry-5)
det(film-2, This-1)
nsubj(tapestry-5, film-2)
cop(tapestry-5, is-3)
det(tapestry-5, a-4)
punct(tapestry-5, ,-6)
det(series-8, a-7)
appos(tapestry-5, series-8)
prep(series-8, of-9)
pobj(of-9, portraits-10)
prep(portraits-10, of-11)
nn(communities-13, Rom-12)
pobj(of-11, communities-13)
partmod(communities-13, woven-14)
advmod(woven-14, together-15)
prep(woven-14, by-16)
pobj(by-16, music-17)
punct(tapestry-5, .-18)

Sentence #2 (33 tokens):
It's very much a musician's film, because of the paucity of spoken dialogue - and what dialogue there is, is not important to the structure of the narrative.
[Text=It CharacterOffsetBegin=99 CharacterOffsetEnd=101 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=101 CharacterOffsetEnd=103 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=104 CharacterOffsetEnd=108 PartOfSpeech=RB] [Text=much CharacterOffsetBegin=109 CharacterOffsetEnd=113 PartOfSpeech=RB] [Text=a CharacterOffsetBegin=114 CharacterOffsetEnd=115 PartOfSpeech=DT] [Text=musician CharacterOffsetBegin=116 CharacterOffsetEnd=124 PartOfSpeech=NN] [Text='s CharacterOffsetBegin=124 CharacterOffsetEnd=126 PartOfSpeech=POS] [Text=film CharacterOffsetBegin=127 CharacterOffsetEnd=131 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=131 CharacterOffsetEnd=132 PartOfSpeech=,] [Text=because CharacterOffsetBegin=133 CharacterOffsetEnd=140 PartOfSpeech=IN] [Text=of CharacterOffsetBegin=141 CharacterOffsetEnd=143 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=144 CharacterOffsetEnd=147 PartOfSpeech=DT] [Text=paucity CharacterOffsetBegin=148 CharacterOffsetEnd=155 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=156 CharacterOffsetEnd=158 PartOfSpeech=IN] [Text=spoken CharacterOffsetBegin=159 CharacterOffsetEnd=165 PartOfSpeech=VBN] [Text=dialogue CharacterOffsetBegin=166 CharacterOffsetEnd=174 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=175 CharacterOffsetEnd=176 PartOfSpeech=:] [Text=and CharacterOffsetBegin=177 CharacterOffsetEnd=180 PartOfSpeech=CC] [Text=what CharacterOffsetBegin=181 CharacterOffsetEnd=185 PartOfSpeech=WP] [Text=dialogue CharacterOffsetBegin=186 CharacterOffsetEnd=194 PartOfSpeech=NN] [Text=there CharacterOffsetBegin=195 CharacterOffsetEnd=200 PartOfSpeech=EX] [Text=is CharacterOffsetBegin=201 CharacterOffsetEnd=203 PartOfSpeech=VBZ] [Text=, CharacterOffsetBegin=203 CharacterOffsetEnd=204 PartOfSpeech=,] [Text=is CharacterOffsetBegin=205 CharacterOffsetEnd=207 PartOfSpeech=VBZ] [Text=not CharacterOffsetBegin=208 CharacterOffsetEnd=211 PartOfSpeech=RB] [Text=important CharacterOffsetBegin=212 CharacterOffsetEnd=221 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=222 CharacterOffsetEnd=224 PartOfSpeech=TO] [Text=the CharacterOffsetBegin=225 CharacterOffsetEnd=228 PartOfSpeech=DT] [Text=structure CharacterOffsetBegin=229 CharacterOffsetEnd=238 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=239 CharacterOffsetEnd=241 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=242 CharacterOffsetEnd=245 PartOfSpeech=DT] [Text=narrative CharacterOffsetBegin=246 CharacterOffsetEnd=255 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=255 CharacterOffsetEnd=256 PartOfSpeech=.] 
root(ROOT-0, film-8)
nsubj(film-8, It-1)
cop(film-8, 's-2)
advmod(much-4, very-3)
advmod(film-8, much-4)
det(musician-6, a-5)
poss(film-8, musician-6)
possessive(musician-6, 's-7)
punct(film-8, ,-9)
mwe(of-11, because-10)
prep(film-8, of-11)
det(paucity-13, the-12)
pobj(of-11, paucity-13)
prep(paucity-13, of-14)
amod(dialogue-16, spoken-15)
pobj(of-14, dialogue-16)
punct(paucity-13, --17)
cc(paucity-13, and-18)
det(dialogue-20, what-19)
dobj(is-22, dialogue-20)
expl(is-22, there-21)
conj(paucity-13, is-22)
punct(film-8, ,-23)
cop(important-26, is-24)
neg(important-26, not-25)
dep(film-8, important-26)
prep(important-26, to-27)
det(structure-29, the-28)
pobj(to-27, structure-29)
prep(structure-29, of-30)
det(narrative-32, the-31)
pobj(of-30, narrative-32)
punct(film-8, .-33)

Sentence #3 (51 tokens):
Some might expect a National Geographic tale of "customs, dress, and music" or a plot-line orbiting a few central characters - don't look for that here.This is because it paints a portrait of a family of peoples, rather than telling a story of individuals.
[Text=Some CharacterOffsetBegin=261 CharacterOffsetEnd=265 PartOfSpeech=DT] [Text=might CharacterOffsetBegin=266 CharacterOffsetEnd=271 PartOfSpeech=MD] [Text=expect CharacterOffsetBegin=272 CharacterOffsetEnd=278 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=279 CharacterOffsetEnd=280 PartOfSpeech=DT] [Text=National CharacterOffsetBegin=281 CharacterOffsetEnd=289 PartOfSpeech=NNP] [Text=Geographic CharacterOffsetBegin=290 CharacterOffsetEnd=300 PartOfSpeech=NNP] [Text=tale CharacterOffsetBegin=301 CharacterOffsetEnd=305 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=306 CharacterOffsetEnd=308 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=309 CharacterOffsetEnd=310 PartOfSpeech=``] [Text=customs CharacterOffsetBegin=310 CharacterOffsetEnd=317 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=317 CharacterOffsetEnd=318 PartOfSpeech=,] [Text=dress CharacterOffsetBegin=319 CharacterOffsetEnd=324 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=324 CharacterOffsetEnd=325 PartOfSpeech=,] [Text=and CharacterOffsetBegin=326 CharacterOffsetEnd=329 PartOfSpeech=CC] [Text=music CharacterOffsetBegin=330 CharacterOffsetEnd=335 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=335 CharacterOffsetEnd=336 PartOfSpeech=''] [Text=or CharacterOffsetBegin=337 CharacterOffsetEnd=339 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=340 CharacterOffsetEnd=341 PartOfSpeech=DT] [Text=plot-line CharacterOffsetBegin=342 CharacterOffsetEnd=351 PartOfSpeech=NN] [Text=orbiting CharacterOffsetBegin=352 CharacterOffsetEnd=360 PartOfSpeech=VBG] [Text=a CharacterOffsetBegin=361 CharacterOffsetEnd=362 PartOfSpeech=DT] [Text=few CharacterOffsetBegin=363 CharacterOffsetEnd=366 PartOfSpeech=JJ] [Text=central CharacterOffsetBegin=367 CharacterOffsetEnd=374 PartOfSpeech=JJ] [Text=characters CharacterOffsetBegin=375 CharacterOffsetEnd=385 PartOfSpeech=NNS] [Text=- CharacterOffsetBegin=386 CharacterOffsetEnd=387 PartOfSpeech=:] [Text=do CharacterOffsetBegin=388 CharacterOffsetEnd=390 PartOfSpeech=VBP] [Text=n't CharacterOffsetBegin=390 CharacterOffsetEnd=393 PartOfSpeech=RB] [Text=look CharacterOffsetBegin=394 CharacterOffsetEnd=398 PartOfSpeech=VB] [Text=for CharacterOffsetBegin=399 CharacterOffsetEnd=402 PartOfSpeech=IN] [Text=that CharacterOffsetBegin=403 CharacterOffsetEnd=407 PartOfSpeech=DT] [Text=here.This CharacterOffsetBegin=408 CharacterOffsetEnd=417 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=418 CharacterOffsetEnd=420 PartOfSpeech=VBZ] [Text=because CharacterOffsetBegin=421 CharacterOffsetEnd=428 PartOfSpeech=IN] [Text=it CharacterOffsetBegin=429 CharacterOffsetEnd=431 PartOfSpeech=PRP] [Text=paints CharacterOffsetBegin=432 CharacterOffsetEnd=438 PartOfSpeech=NNS] [Text=a CharacterOffsetBegin=439 CharacterOffsetEnd=440 PartOfSpeech=DT] [Text=portrait CharacterOffsetBegin=441 CharacterOffsetEnd=449 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=450 CharacterOffsetEnd=452 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=453 CharacterOffsetEnd=454 PartOfSpeech=DT] [Text=family CharacterOffsetBegin=455 CharacterOffsetEnd=461 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=462 CharacterOffsetEnd=464 PartOfSpeech=IN] [Text=peoples CharacterOffsetBegin=465 CharacterOffsetEnd=472 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=472 CharacterOffsetEnd=473 PartOfSpeech=,] [Text=rather CharacterOffsetBegin=474 CharacterOffsetEnd=480 PartOfSpeech=RB] [Text=than CharacterOffsetBegin=481 CharacterOffsetEnd=485 PartOfSpeech=IN] [Text=telling CharacterOffsetBegin=486 CharacterOffsetEnd=493 PartOfSpeech=VBG] [Text=a CharacterOffsetBegin=494 CharacterOffsetEnd=495 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=496 CharacterOffsetEnd=501 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=502 CharacterOffsetEnd=504 PartOfSpeech=IN] [Text=individuals CharacterOffsetBegin=505 CharacterOffsetEnd=516 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=516 CharacterOffsetEnd=517 PartOfSpeech=.] 
root(ROOT-0, expect-3)
nsubj(expect-3, Some-1)
aux(expect-3, might-2)
det(tale-7, a-4)
nn(tale-7, National-5)
nn(tale-7, Geographic-6)
dobj(expect-3, tale-7)
prep(tale-7, of-8)
punct(of-8, ``-9)
pobj(of-8, customs-10)
punct(customs-10, ,-11)
conj(customs-10, dress-12)
punct(customs-10, ,-13)
cc(customs-10, and-14)
conj(customs-10, music-15)
punct(of-8, ''-16)
cc(tale-7, or-17)
nsubj(is-32, a-18)
amod(a-18, plot-line-19)
partmod(plot-line-19, orbiting-20)
det(characters-24, a-21)
amod(characters-24, few-22)
amod(characters-24, central-23)
nsubj(look-28, characters-24)
punct(characters-24, --25)
aux(look-28, do-26)
neg(look-28, n't-27)
ccomp(orbiting-20, look-28)
prep(look-28, for-29)
det(here.This-31, that-30)
pobj(for-29, here.This-31)
conj(tale-7, is-32)
mark(paints-35, because-33)
nsubj(paints-35, it-34)
advcl(is-32, paints-35)
det(portrait-37, a-36)
dobj(paints-35, portrait-37)
prep(portrait-37, of-38)
det(family-40, a-39)
pobj(of-38, family-40)
prep(family-40, of-41)
pobj(of-41, peoples-42)
punct(paints-35, ,-43)
cc(paints-35, rather-44)
mwe(rather-44, than-45)
conj(paints-35, telling-46)
det(story-48, a-47)
dobj(telling-46, story-48)
prep(story-48, of-49)
pobj(of-49, individuals-50)
punct(expect-3, .-51)

Sentence #4 (25 tokens):
The plot is the story through space (India to Andalucia) and/or/ time (we cannot tell) of a people.
[Text=The CharacterOffsetBegin=522 CharacterOffsetEnd=525 PartOfSpeech=DT] [Text=plot CharacterOffsetBegin=526 CharacterOffsetEnd=530 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=531 CharacterOffsetEnd=533 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=534 CharacterOffsetEnd=537 PartOfSpeech=DT] [Text=story CharacterOffsetBegin=538 CharacterOffsetEnd=543 PartOfSpeech=NN] [Text=through CharacterOffsetBegin=544 CharacterOffsetEnd=551 PartOfSpeech=IN] [Text=space CharacterOffsetBegin=552 CharacterOffsetEnd=557 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=558 CharacterOffsetEnd=559 PartOfSpeech=-LRB-] [Text=India CharacterOffsetBegin=559 CharacterOffsetEnd=564 PartOfSpeech=NNP] [Text=to CharacterOffsetBegin=565 CharacterOffsetEnd=567 PartOfSpeech=TO] [Text=Andalucia CharacterOffsetBegin=568 CharacterOffsetEnd=577 PartOfSpeech=NNP] [Text=-RRB- CharacterOffsetBegin=577 CharacterOffsetEnd=578 PartOfSpeech=-RRB-] [Text=and/or CharacterOffsetBegin=579 CharacterOffsetEnd=585 PartOfSpeech=CC] [Text=/ CharacterOffsetBegin=585 CharacterOffsetEnd=586 PartOfSpeech=:] [Text=time CharacterOffsetBegin=587 CharacterOffsetEnd=591 PartOfSpeech=NN] [Text=-LRB- CharacterOffsetBegin=592 CharacterOffsetEnd=593 PartOfSpeech=-LRB-] [Text=we CharacterOffsetBegin=593 CharacterOffsetEnd=595 PartOfSpeech=PRP] [Text=can CharacterOffsetBegin=596 CharacterOffsetEnd=599 PartOfSpeech=MD] [Text=not CharacterOffsetBegin=599 CharacterOffsetEnd=602 PartOfSpeech=RB] [Text=tell CharacterOffsetBegin=603 CharacterOffsetEnd=607 PartOfSpeech=VB] [Text=-RRB- CharacterOffsetBegin=607 CharacterOffsetEnd=608 PartOfSpeech=-RRB-] [Text=of CharacterOffsetBegin=609 CharacterOffsetEnd=611 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=612 CharacterOffsetEnd=613 PartOfSpeech=DT] [Text=people CharacterOffsetBegin=614 CharacterOffsetEnd=620 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=620 CharacterOffsetEnd=621 PartOfSpeech=.] 
root(ROOT-0, story-5)
det(plot-2, The-1)
nsubj(story-5, plot-2)
cop(story-5, is-3)
det(story-5, the-4)
prep(story-5, through-6)
nn(India-9, space-7)
punct(India-9, -LRB--8)
pobj(through-6, India-9)
prep(story-5, to-10)
pobj(to-10, Andalucia-11)
punct(story-5, -RRB--12)
cc(story-5, and/or-13)
punct(story-5, /-14)
dep(story-5, time-15)
punct(tell-20, -LRB--16)
nsubj(tell-20, we-17)
aux(tell-20, can-18)
neg(tell-20, not-19)
dep(time-15, tell-20)
punct(tell-20, -RRB--21)
prep(tell-20, of-22)
det(people-24, a-23)
pobj(of-22, people-24)
punct(story-5, .-25)

Sentence #5 (7 tokens):
There is no need of narration.
[Text=There CharacterOffsetBegin=626 CharacterOffsetEnd=631 PartOfSpeech=EX] [Text=is CharacterOffsetBegin=632 CharacterOffsetEnd=634 PartOfSpeech=VBZ] [Text=no CharacterOffsetBegin=635 CharacterOffsetEnd=637 PartOfSpeech=DT] [Text=need CharacterOffsetBegin=638 CharacterOffsetEnd=642 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=643 CharacterOffsetEnd=645 PartOfSpeech=IN] [Text=narration CharacterOffsetBegin=646 CharacterOffsetEnd=655 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=655 CharacterOffsetEnd=656 PartOfSpeech=.] 
root(ROOT-0, is-2)
expl(is-2, There-1)
det(need-4, no-3)
nsubj(is-2, need-4)
prep(need-4, of-5)
pobj(of-5, narration-6)
punct(is-2, .-7)

Sentence #6 (21 tokens):
You get a sense of a joyous people, strongly linked in small communities where social interaction is very important.
[Text=You CharacterOffsetBegin=661 CharacterOffsetEnd=664 PartOfSpeech=PRP] [Text=get CharacterOffsetBegin=665 CharacterOffsetEnd=668 PartOfSpeech=VBP] [Text=a CharacterOffsetBegin=669 CharacterOffsetEnd=670 PartOfSpeech=DT] [Text=sense CharacterOffsetBegin=671 CharacterOffsetEnd=676 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=677 CharacterOffsetEnd=679 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=680 CharacterOffsetEnd=681 PartOfSpeech=DT] [Text=joyous CharacterOffsetBegin=682 CharacterOffsetEnd=688 PartOfSpeech=JJ] [Text=people CharacterOffsetBegin=689 CharacterOffsetEnd=695 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=695 CharacterOffsetEnd=696 PartOfSpeech=,] [Text=strongly CharacterOffsetBegin=697 CharacterOffsetEnd=705 PartOfSpeech=RB] [Text=linked CharacterOffsetBegin=706 CharacterOffsetEnd=712 PartOfSpeech=VBN] [Text=in CharacterOffsetBegin=713 CharacterOffsetEnd=715 PartOfSpeech=IN] [Text=small CharacterOffsetBegin=716 CharacterOffsetEnd=721 PartOfSpeech=JJ] [Text=communities CharacterOffsetBegin=722 CharacterOffsetEnd=733 PartOfSpeech=NNS] [Text=where CharacterOffsetBegin=734 CharacterOffsetEnd=739 PartOfSpeech=WRB] [Text=social CharacterOffsetBegin=740 CharacterOffsetEnd=746 PartOfSpeech=JJ] [Text=interaction CharacterOffsetBegin=747 CharacterOffsetEnd=758 PartOfSpeech=NN] [Text=is CharacterOffsetBegin=759 CharacterOffsetEnd=761 PartOfSpeech=VBZ] [Text=very CharacterOffsetBegin=762 CharacterOffsetEnd=766 PartOfSpeech=RB] [Text=important CharacterOffsetBegin=767 CharacterOffsetEnd=776 PartOfSpeech=JJ] [Text=. CharacterOffsetBegin=776 CharacterOffsetEnd=777 PartOfSpeech=.] 
root(ROOT-0, get-2)
nsubj(get-2, You-1)
det(sense-4, a-3)
dobj(get-2, sense-4)
prep(sense-4, of-5)
det(people-8, a-6)
amod(people-8, joyous-7)
pobj(of-5, people-8)
punct(get-2, ,-9)
advmod(linked-11, strongly-10)
dep(get-2, linked-11)
prep(linked-11, in-12)
amod(communities-14, small-13)
pobj(in-12, communities-14)
advmod(important-20, where-15)
amod(interaction-17, social-16)
nsubj(important-20, interaction-17)
cop(important-20, is-18)
advmod(important-20, very-19)
rcmod(communities-14, important-20)
punct(get-2, .-21)

Sentence #7 (55 tokens):
And a great sense of sadness in parts, at their rejection by society at large.So it's a paean to Rom culture, very beautifully shot, with a wide spectrum of Rom music, and a sting in the tail which is the oppression these people have faced, and still face.
[Text=And CharacterOffsetBegin=782 CharacterOffsetEnd=785 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=786 CharacterOffsetEnd=787 PartOfSpeech=DT] [Text=great CharacterOffsetBegin=788 CharacterOffsetEnd=793 PartOfSpeech=JJ] [Text=sense CharacterOffsetBegin=794 CharacterOffsetEnd=799 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=800 CharacterOffsetEnd=802 PartOfSpeech=IN] [Text=sadness CharacterOffsetBegin=803 CharacterOffsetEnd=810 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=811 CharacterOffsetEnd=813 PartOfSpeech=IN] [Text=parts CharacterOffsetBegin=814 CharacterOffsetEnd=819 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=819 CharacterOffsetEnd=820 PartOfSpeech=,] [Text=at CharacterOffsetBegin=821 CharacterOffsetEnd=823 PartOfSpeech=IN] [Text=their CharacterOffsetBegin=824 CharacterOffsetEnd=829 PartOfSpeech=PRP$] [Text=rejection CharacterOffsetBegin=830 CharacterOffsetEnd=839 PartOfSpeech=NN] [Text=by CharacterOffsetBegin=840 CharacterOffsetEnd=842 PartOfSpeech=IN] [Text=society CharacterOffsetBegin=843 CharacterOffsetEnd=850 PartOfSpeech=NN] [Text=at CharacterOffsetBegin=851 CharacterOffsetEnd=853 PartOfSpeech=IN] [Text=large.So CharacterOffsetBegin=854 CharacterOffsetEnd=862 PartOfSpeech=NNP] [Text=it CharacterOffsetBegin=863 CharacterOffsetEnd=865 PartOfSpeech=PRP] [Text='s CharacterOffsetBegin=865 CharacterOffsetEnd=867 PartOfSpeech=VBZ] [Text=a CharacterOffsetBegin=868 CharacterOffsetEnd=869 PartOfSpeech=DT] [Text=paean CharacterOffsetBegin=870 CharacterOffsetEnd=875 PartOfSpeech=NN] [Text=to CharacterOffsetBegin=876 CharacterOffsetEnd=878 PartOfSpeech=TO] [Text=Rom CharacterOffsetBegin=879 CharacterOffsetEnd=882 PartOfSpeech=NN] [Text=culture CharacterOffsetBegin=883 CharacterOffsetEnd=890 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=890 CharacterOffsetEnd=891 PartOfSpeech=,] [Text=very CharacterOffsetBegin=892 CharacterOffsetEnd=896 PartOfSpeech=RB] [Text=beautifully CharacterOffsetBegin=897 CharacterOffsetEnd=908 PartOfSpeech=RB] [Text=shot CharacterOffsetBegin=909 CharacterOffsetEnd=913 PartOfSpeech=VBN] [Text=, CharacterOffsetBegin=913 CharacterOffsetEnd=914 PartOfSpeech=,] [Text=with CharacterOffsetBegin=915 CharacterOffsetEnd=919 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=920 CharacterOffsetEnd=921 PartOfSpeech=DT] [Text=wide CharacterOffsetBegin=922 CharacterOffsetEnd=926 PartOfSpeech=JJ] [Text=spectrum CharacterOffsetBegin=927 CharacterOffsetEnd=935 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=936 CharacterOffsetEnd=938 PartOfSpeech=IN] [Text=Rom CharacterOffsetBegin=939 CharacterOffsetEnd=942 PartOfSpeech=NN] [Text=music CharacterOffsetBegin=943 CharacterOffsetEnd=948 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=948 CharacterOffsetEnd=949 PartOfSpeech=,] [Text=and CharacterOffsetBegin=950 CharacterOffsetEnd=953 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=954 CharacterOffsetEnd=955 PartOfSpeech=DT] [Text=sting CharacterOffsetBegin=956 CharacterOffsetEnd=961 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=962 CharacterOffsetEnd=964 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=965 CharacterOffsetEnd=968 PartOfSpeech=DT] [Text=tail CharacterOffsetBegin=969 CharacterOffsetEnd=973 PartOfSpeech=NN] [Text=which CharacterOffsetBegin=974 CharacterOffsetEnd=979 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=980 CharacterOffsetEnd=982 PartOfSpeech=VBZ] [Text=the CharacterOffsetBegin=983 CharacterOffsetEnd=986 PartOfSpeech=DT] [Text=oppression CharacterOffsetBegin=987 CharacterOffsetEnd=997 PartOfSpeech=NN] [Text=these CharacterOffsetBegin=998 CharacterOffsetEnd=1003 PartOfSpeech=DT] [Text=people CharacterOffsetBegin=1004 CharacterOffsetEnd=1010 PartOfSpeech=NNS] [Text=have CharacterOffsetBegin=1011 CharacterOffsetEnd=1015 PartOfSpeech=VBP] [Text=faced CharacterOffsetBegin=1016 CharacterOffsetEnd=1021 PartOfSpeech=VBN] [Text=, CharacterOffsetBegin=1021 CharacterOffsetEnd=1022 PartOfSpeech=,] [Text=and CharacterOffsetBegin=1023 CharacterOffsetEnd=1026 PartOfSpeech=CC] [Text=still CharacterOffsetBegin=1027 CharacterOffsetEnd=1032 PartOfSpeech=RB] [Text=face CharacterOffsetBegin=1033 CharacterOffsetEnd=1037 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=1037 CharacterOffsetEnd=1038 PartOfSpeech=.] 
root(ROOT-0, And-1)
det(sense-4, a-2)
amod(sense-4, great-3)
dep(And-1, sense-4)
prep(sense-4, of-5)
pobj(of-5, sadness-6)
prep(sadness-6, in-7)
pobj(in-7, parts-8)
punct(sense-4, ,-9)
prep(sense-4, at-10)
poss(rejection-12, their-11)
pobj(at-10, rejection-12)
prep(rejection-12, by-13)
pobj(by-13, society-14)
prep(rejection-12, at-15)
pobj(at-15, large.So-16)
nsubj(paean-20, it-17)
cop(paean-20, 's-18)
det(paean-20, a-19)
rcmod(large.So-16, paean-20)
prep(paean-20, to-21)
nn(culture-23, Rom-22)
pobj(to-21, culture-23)
punct(sense-4, ,-24)
advmod(beautifully-26, very-25)
advmod(shot-27, beautifully-26)
partmod(sense-4, shot-27)
punct(shot-27, ,-28)
prep(shot-27, with-29)
det(spectrum-32, a-30)
amod(spectrum-32, wide-31)
pobj(with-29, spectrum-32)
prep(spectrum-32, of-33)
nn(music-35, Rom-34)
pobj(of-33, music-35)
punct(sense-4, ,-36)
cc(sense-4, and-37)
det(sting-39, a-38)
conj(sense-4, sting-39)
prep(sting-39, in-40)
det(tail-42, the-41)
pobj(in-40, tail-42)
nsubj(oppression-46, which-43)
cop(oppression-46, is-44)
det(oppression-46, the-45)
rcmod(tail-42, oppression-46)
det(people-48, these-47)
nsubj(faced-50, people-48)
aux(faced-50, have-49)
rcmod(oppression-46, faced-50)
punct(sense-4, ,-51)
cc(sense-4, and-52)
advmod(sense-4, still-53)
conj(sense-4, face-54)
punct(And-1, .-55)

